<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TvShowWidget</class>
 <widget class="QWidget" name="TvShowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>443</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tabEpisode">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Episode</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>437</width>
            <height>486</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::ExpandingFieldsGrow</enum>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="labelEpisodeTitle">
             <property name="text">
              <string>Title:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="KLineEdit" name="editEpisodeTitle"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelEpisodeNumber">
             <property name="text">
              <string>Number:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="KIntNumInput" name="editEpisodeNumber"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="labelEpisodeFirstAired">
             <property name="text">
              <string>First Aired:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="KLineEdit" name="editEpisodeFirstAired"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="labelEpisodeGenres">
             <property name="text">
              <string>Genres:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="KLineEdit" name="editEpisodeGenres"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="labelEpisodeDirector">
             <property name="text">
              <string>Director:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="KLineEdit" name="editEpisodeDirector"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="labelEpisodeWriter">
             <property name="text">
              <string>Writer:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="KLineEdit" name="editEpisodeWriter"/>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="labelEpisodeActors">
             <property name="text">
              <string>Actors:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="KLineEdit" name="editEpisodeActors"/>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="labelEpisodeSeeAlso">
             <property name="text">
              <string>See Also:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="KLineEdit" name="editEpisodeSeeAlso"/>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="labelEpisodeBanner">
             <property name="text">
              <string>Banner:</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="KLineEdit" name="editEpisodeBanner"/>
           </item>
           <item row="9" column="1">
            <widget class="KLineEdit" name="editEpisodePoster"/>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="labelEpisodePoster">
             <property name="text">
              <string>Poster:</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="labelEpisodeSynopsis">
             <property name="text">
              <string>Synopsis:</string>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="KTextEdit" name="editEpisodeSynopsis">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSeason">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Season</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>178</width>
            <height>83</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="labelSeasonNumber">
             <property name="text">
              <string>Number:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="KIntNumInput" name="editSeasonNumber"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelSeasonBanner">
             <property name="text">
              <string>Banner:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="KLineEdit" name="editSeasonBanner"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelSeasonPoster">
             <property name="text">
              <string>Poster:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="KLineEdit" name="editSeasonPoster"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabShow">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Show</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>446</width>
            <height>406</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QFormLayout" name="formLayout_3">
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="labelShowTitle">
             <property name="text">
              <string>Title:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="KLineEdit" name="editShowTitle"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelShowBanner">
             <property name="text">
              <string>Banner:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="KLineEdit" name="editShowBanner"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="labelShowPoster">
             <property name="text">
              <string>Poster:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="KLineEdit" name="editShowPoster"/>
           </item>
           <item row="4" column="1">
            <widget class="KTextEdit" name="editShowSynopsis"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="labelShowSynopsis">
             <property name="text">
              <string>Synopsis:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
